os/mm/mm_heap/mm_initialize.c

```diff
@@ -127,6 +127,13 @@ void mm_addregion(FAR struct mm_heap_s *heap, FAR void *heapstart, size_t heapsi
 
 	heapbase = MM_ALIGN_UP((uintptr_t)heapstart);
 	heapend  = MM_ALIGN_DOWN((uintptr_t)heapstart + (uintptr_t)heapsize);
```

Comment:
At line 111, heapsize should be checked whether it is non-zero positive values, for example, if (heapsize <= 0) return;

Reply:
Modified. Thanks.

Reply:
I reverted. Because `heapsize` is `size_t` type, and `size_t` is unsigned int. So it cannot be negative.